Envision is missing packages

## Describe the bug

I've tried using [Envision](https://search.nixos.org/packages?channel=24.05&show=envision&from=0&size=50&sort=relevance&type=packages&query=envision) and wasn't able to proceed further due to missing packages warning. Same on unwrapped version, but with more missing packages

## Steps To Reproduce

Steps to reproduce the behavior:

1. $ nix-shell -p envision --run envision
2. Press "Build profile"
 
## Expected behavior

Generation of VR profile for Valve Index

## Screenshots

![image](https://github.com/user-attachments/assets/1d0dd7ce-4d5b-4105-a064-3859bb1345e4)

## Additional context

Hyprland + Framework 16 (tried both expansion card slots and direct connection to GPU)

## Metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.62, NixOS, 25.05 (Warbler), 25.05.20241119.23e89b7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.10`
 - nixpkgs: `/nix/store/c9wv7i0af6mysmy65x6nvyfw5izzxv4g-source`

## Notify maintainers

@Pandapip1, @Scrumplex

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

The changes in #372276 should be on unstable now according to https://nixpk.gs/pr-tracker.html?pr=372276 but I'm still encountering the same libnotify issue when attempting to build wivrn.

<details>

<summary>envision build log</summary>

```
Building WiVRn...
Already on 'release'
Your branch is up to date with 'origin/release'.
Already up to date.
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.2")
-- Found Vulkan: /usr/lib/libvulkan.so (found version "1.3.296") found components: glslc glslangValidator
-- Found Git: /usr/bin/git (found version "2.47.0")
-- Found OpenSSL: /usr/lib/libcrypto.so (found version "3.3.2")
CMake Warning (dev) at CMakeLists.txt:83 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Boost: /usr/lib64/cmake/Boost-1.86.0/BoostConfig.cmake (found suitable version "1.86.0", minimum required is "1.75.0") found components: locale
-- Checking for modules 'libavcodec;libavutil;libswscale;libavfilter'
--   Found libavcodec, version 61.19.100
--   Found libavutil, version 59.39.100
--   Found libswscale, version 8.3.100
--   Found libavfilter, version 10.4.100
-- Checking for module 'libdrm'
--   Found libdrm, version 2.4.123
-- Checking for module 'x264'
--   Found x264, version 0.164.x
-- Checking for module 'libsystemd'
--   Found libsystemd, version 256
-- Checking for module 'libpipewire-0.3'
--   Found libpipewire-0.3, version 1.2.7
-- Checking for module 'libpulse'
--   Found libpulse, version 17.0
-- Checking for modules 'avahi-client;avahi-glib'
--   Found avahi-client, version 0.8
--   Found avahi-glib, version 0.8
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Found nlohmann_json: /usr/share/cmake/nlohmann_json/nlohmann_jsonConfig.cmake (found version "3.11.3")
-- Checking for modules 'glib-2.0;gio-2.0;gio-unix-2.0'
--   Found glib-2.0, version 2.82.1
--   Found gio-2.0, version 2.82.1
--   Found gio-unix-2.0, version 2.82.1
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Checking for module 'libnotify'
--   Package 'gdk-pixbuf-2.0', required by 'libnotify', not found
CMake Error at /nix/store/38cffsqqx823crf1i4bcf6zz1qz1hgpd-cmake-3.30.5/share/cmake-3.30/Modules/FindPkgConfig.cmake:645 (message):
  The following required packages were not found:

   - libnotify

Call Stack (most recent call first):
  /nix/store/38cffsqqx823crf1i4bcf6zz1qz1hgpd-cmake-3.30.5/share/cmake-3.30/Modules/FindPkgConfig.cmake:873 (_pkg_check_modules_internal)
  CMakeLists.txt:179 (pkg_check_modules)


-- Configuring incomplete, errors occurred!

```

</details>

--- a/pkgs/by-name/en/envision-unwrapped/package.nix
+++ b/pkgs/by-name/en/envision-unwrapped/package.nix
@@ -106,6 +106,9 @@ stdenv.mkDerivation (finalAttrs: {
     homepage = "https://gitlab.com/gabmus/envision";
     license = lib.licenses.agpl3Only;
     mainProgram = "envision";
+    # More maintainers needed!
+    # envision (wrapped) requires frequent updates to the dependency list;
+    # the more people that can help with this, the better.
     maintainers = with lib.maintainers; [
       pandapip1
       Scrumplex

--- a/pkgs/by-name/en/envision/package.nix
+++ b/pkgs/by-name/en/envision/package.nix
@@ -61,6 +61,7 @@ buildFHSEnv {
           gtest
           jq
           libepoxy
+          lz4
           lz4.dev
           tbb
           libxkbcommon

--- a/pkgs/by-name/en/envision/package.nix
+++ b/pkgs/by-name/en/envision/package.nix
@@ -52,6 +52,7 @@ buildFHSEnv {
           xorg.xorgproto
           SDL2
           wayland
+          mesa-gl-headers
           # Additional dependencies required for Monado WMR support
           bc
           fmt

